R/parser.R

```diff
@@ -819,7 +819,14 @@ parse_mbox <- function(perceval_path,mbox_path){
 #' @family parsers
 parse_jira <- function(json_path){
 
-  json_issue_comments <- jsonlite::read_json(json_path)
+  if (json_path == "../../rawdata/issue_tracker/geronimo_issues.json") {
```

Comment:
Be aware that the path "../../rawdata/issue_tracker/geronimo_issues.json" is just the default path in geronmo.yml. The user could very much choose to change this and then this check will not work. Or the user may use a different config file completely